src/app/components/ClientCard/OneTimeLinkBtn.vue

```diff
@@ -12,6 +12,12 @@
 const props = defineProps<{ client: LocalClient }>();
 
 const clientsStore = useClientsStore();
+const toast = useToast();
+const { copy, copied, isSupported } = useClipboard();
+const oneTimeLink = computed(
+  () =>
+    `${document.location.protocol}//${document.location.host}/cnf/${props.client.oneTimeLink.oneTimeLink}`
```

Comment:
this link is now used in multiple places. maybe refactoring to avoid duplication makes sense